### PR TITLE
Revamp

### PR DESCRIPTION
**Pull request by jmagrippis**

### CODE REVIEW

File: src/BlockGrid.test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+describe('BlockGrid', () => {
+  xit('should be implemented!', () => {});
```
Comment (labithiotis) [inline src/BlockGrid.test.js]: maybe just add one test that says render grid of 10x10 or something.

**Comment by jmagrippis** [comment]
> once we're happy with this we should collapse all the commits to just a single initial commit.

You know GitHub let's you squash and merge automatically, right? 😁

**Comment by labithiotis** [comment]
@jmagrippis But you still see commits in master (I know its just one with all the comment messages in one). I'm just meant we should collapse all commits, not just this PR.

**Comment by jmagrippis** [comment]
@labithiotis Ah, I see what you mean! Like, give the candidates a "fresh" repo, with just the one commit that sets everything up?

I don't know, I don't think it'll hurt if they see we put in a couple touch-up PRs over the years!

**Comment by labithiotis** [comment]
@jmagrippis Yep, a fresh repo. One of the reasons we have rejigged is to give off a better impression and think a clean/fresh repo is towards that end too.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
```
Comment (mjgreen145) [inline src/index.css]: Let's finally remove duplicate margin and padding...

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  background: grey;
+  float: left;
+  width: 10%;
+  height: 100%;
+}
+
+.block {
+  width: 100%;
+  height: 10%;
+  background: red;
```
Comment (mjgreen145) [inline src/index.css]: why does this have a red background if all blocks get a color?

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+#grid {
+  width: 100%;
+  height: 100%;
+  background: grey;
+}
+
+.col {
+  background: grey;
```
Comment (mjgreen145) [inline src/index.css]: This will inherit from the #grid

**Pull request merged**